Automattic/newspack-plugin
# Site Design Wizard: Theme Mod Controls

### All Submissions:






### Changes proposed in this Pull Request:

Adds a 2nd tab to the Site Design Wizard with the Theme Options.

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Navigate to Site Design > Settings
3. Change the different options and check if they are being applied.
4. Check the Customizer if the 2 matches.

### Other information:
